core/lib/dal/migrations/20231212121822_add_miniblocks_fee_account_address.up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE miniblocks
+    ADD COLUMN fee_account_address BYTEA NOT NULL DEFAULT '\x0000000000000000000000000000000000000000'::bytea;
```

Comment:
I'll need to check whether there are other RPC methods returning this data, but the obvious one is `zks_getBlockDetails`. Would it be OK for it to return an error in case of non-migrated address (i.e., if `fee_account_address == Address::zero()`)? Another option would be to access `l1_batches.fee_account_address` in this case; it *should* continue working even after this column is removed because this case would never be triggered. I don't really like this non-local / cumbersome reasoning though.

Reply:
Returning error will result in indexers' halt during the migration, as I understand its time is of order of 2-3 minutes, so doesn't sound like a good thing.
If implementing a fallback to `l1_batches.fee_account_address` is not really difficult, I'd go this way.